packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_def.hpp

```diff
@@ -51,77 +51,127 @@ namespace FROSch {
     using namespace Xpetra;
 
     template <class SC,class LO,class GO,class NO>
-    CoarseSpace<SC,LO,GO,NO>::CoarseSpace() :
-    SerialRowMap_ (),
+    CoarseSpace<SC,LO,GO,NO>::CoarseSpace(CommPtr mpiComm,
+                                          CommPtr serialComm) :
+    MpiComm_ (mpiComm),
+    SerialComm_ (serialComm),
     UnassembledBasesMaps_ (0),
+    UnassembledBasesMapsUnique_ (0),
     UnassembledSubspaceBases_ (0),
+    Offsets_ (0),
     AssembledBasisMap_ (),
```

Comment:
You don't need to initialize the RCPs here; they get initialized to `nullptr` by default.

Reply:
see above.

Reply:
See above.

Reply:
@searhein Also see above.  You've added lines of code that do nothing.  They are also more prone to warnings like out-of-order initialization, if some other developer changes the order of members.